fleet_vehicle_service_calendar/views/fleet_vehicle_log_services_views.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="fleet_vehicle_log_services_view_form" model="ir.ui.view">
+        <field name="name">fleet.vehicle.log.services.form.calendar</field>
+        <field name="model">fleet.vehicle.log.services</field>
+        <field
+            name="inherit_id"
+            ref="fleet_vehicle_service_kanban.fleet_vehicle_log_services_view_form"
+        />
+        <field name="arch" type="xml">
+            <div name="button_box" position="inside">
+                <button
```

Comment:
The `attrs` added in the div `button_box` in fleet_vehicle_service_kanban is hiding the meetings button, can you fix that please ? also review the button `toggle_active` in that module, I think it is not working.
https://github.com/OCA/fleet/blob/4b1c8b4ce7b192aa12f87baa2894688789c35963/fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml#L19

Reply:
Fixed the meeting button, give it a look.

Reply:
@mymage , it's an issue with `fleet_vehicle_service_kanban`.

The button box should have this invisible attribute. Moreover, even if you remove the `attrs` and make it visible, the toggle_active button looks ugly because it has some css classes that it shouldn't have:

![image](https://user-images.githubusercontent.com/1914185/85108323-cbb34b00-b20f-11ea-955b-5275a6fe2612.png)


These things need to be fixed in the `fleet_vehicle_service_kanban` module, ideally in another PR, before continuing the work on this one.


To be clear, this is how a button box should look like:
![image](https://user-images.githubusercontent.com/1914185/85108356-dd94ee00-b20f-11ea-889d-0ed4cbace193.png)